docs/how/updating-datahub.md

```diff
@@ -23,14 +25,21 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 - #13004: The `acryl-datahub-airflow-plugin` dropped support for Airflow 2.3 and 2.4.
 - #13186: NoCode Migration Removed - This code hasn't been required in many years. If needed, a user should upgrade to DataHub 1.0.x prior to upgrading to a later version.
 
+### Known Issues
+
+- #13397 - ASYNC_BLOCKING/ASYNC_NONBLOCKING - Async modes are impacted by kafka lag.
+
 ### Potential Downtime
 
 ### Deprecations
 
 ### Other Notable Changes
 
 - #13165 - OpenAPI v3 Patching Improvements
-- #13349 - Python SDK to use OpenAPI (sync & async) and API Tracing (async) writes by default.
+- #13397 - Ingestion Rest Emitter - Added EmitMode parameter for write guarantees.
```

Comment:
we should also call out that `async_flag` was removed as a breaking change. I think that breaking change is totally fine to do since idt people were using it